### ISSUE TITLE
Rename "New Molecule" button to "Reset Molecule" button

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#265]
This is a design change to make it more it more clear that the molecule (Ozone or Nitrogen Dioxide) were destroyed when they broke apart. Learners are putting a new molecule in the observation window and not "returning" a repaired one.
 
This issue is related to https://github.com/phetsims/molecules-and-light/issues/252 identifying that the timing of the button's context response has to be tied to the actual appearance of the button.

**Comment by terracoda** [comment]
From #252, 
>> the context response offers a help-text-like interaction hint: "Look for New Molecule button."

**Comment by jessegreenberg** [comment]
Since this changes a visual (translated) string, we will have to consider translations. A change to the string will impact the english simulation only. If it is important that it apply to the translated versions we will need to notify translators to re-translate when this sim is republished. @arouinfar @terracoda is that something we should do?

**Comment by jessegreenberg** [comment]
Almost done for english with the above commit. For a11y, we also have this string.
```js
    moleculesOutOfViewPatternString: {
      value: '{{firstMolecule}} and {{secondMolecule}} out of view. Return molecule to make more observations.'
    },
```

Should this change to "... Create new molecule to make more observations."?

**Comment by terracoda** [comment]
I like I mentioned, "Add New Molecule..." in the design doc, but I like, "Create New Molecule..." better.

**Comment by terracoda** [comment]
Actually, @jessegreenberg, is this a context response? Or a string in the Screen summary?

@Matthew-phet and I thought, "Look for New Molecule button." would work as a context response.

**Comment by jessegreenberg** [comment]
> I think I mentioned, "Add new molecule..." in the design doc,

Oh sorry, I missed it!

>  is this a context response? Or a string in the Screen summary?

This is the description for the Observation Window in this case
![image](https://user-images.githubusercontent.com/6396244/73768310-c9f85780-4746-11ea-987d-62c0f56bfaeb.png)

**Comment by terracoda** [comment]
Ah! Thanks. Maybe stick with, "Add new molecule to make more observations." It's simple, and there is more than one way to add a molecule - the button and the radio buttons.

**Comment by arouinfar** [comment]
@jessegreenberg the string on the button should be "New Molecule" so it aligns with the descriptions. The translated versions will then be out-of-sync with the English version and the updated string should probably be re-translated. I think @oliver-phet would be the one to handle contacting the translators about something like this.

**Comment by terracoda** [comment]
@jessegreenberg, I'm not sure an example of the Observation Window you posted in https://github.com/phetsims/molecules-and-light/issues/265#issuecomment-582013486 is actually in the design doc, so you didn't miss anything ;-) You found something.

**Comment by terracoda** [comment]
Assigning back to you @jessegreenberg to change the label of the button.
I think these are the three things we want:
1. new label for button: 
  - "New Molecule"
2. new context response tied to the appearance of the button: 
  - "Look for New Molecule button."
3. any interaction hints that are needed in the state descriptions to help learners figure out that they need to put a molecule back into the observation window, should be general because there is more than one way to do that
  - "Add new molecule to make more observations."

We could consider removing "new" from number 3, but I think it is just fine for now.

**Comment by jessegreenberg** [comment]
1 and 2 from the above comment are done. Sorry, I don't understand 3. Should there be a new paragraph with that in the screen summary? I don't see that paragraph in the design document.

**Comment by jessegreenberg** [comment]
From meeting on 2/25/19 we are going to change again to "Return Molecule" button. There was confusion in a user interview that "New" molecule meant that we were adding a different kind molecule to the observation window, or adding a new molecule to the radio button list.

**Comment by terracoda** [comment]
@jessegreenberg, we might need 3 updates to address the "Reset Molecule" button:
- update the button's label: "Reset Molecule"
- update the response delivered when button appears: "Look for Reset Molecule button."
- update the interaction his in the Observation Window that comes after the Split molecule description:
  - NO and O out of view. Reset molecule to make more observations.

**Comment by terracoda** [comment]
Assigning to @Matthew-Moore240 and @jessegreenberg, think the above is what we agreed to in our meeting yesterday.

@jessegreenberg, I think it is safe to implement, @Matthew-Moore240 please comment. My main concern is if the string in number 4 is the best?

**Comment by jessegreenberg** [comment]
Thanks for the reminder @terracoda. Rename done in all 4 places in the above commits. IMO your suggestion for #4 sounds great for consistency. Assigning to @Matthew-Moore240 for review.

**Comment by Matthew-Moore240** [comment]
Yup it all looks good to me!

**Comment by terracoda** [comment]
This is all done. Closing.
